google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1/JsonStreamWriterTest.java

```diff
@@ -603,4 +607,45 @@ public void testFlowControlSettingNoLimitBehavior() throws Exception {
       appendFuture.get();
     }
   }
+
+  @Test
+  public void testMultipleAppendSerializtionErrors()
```

Comment:
I am not sure about the convention of this file, but commonly this would be split into separate unit tests?